oss.gg/7_create_a_template.md

```
@@ -38,14 +37,14 @@ Your turn 👇
 » 10-October-2024 by Novin Noori [Protect and Rate Limit Rust + Actix Web API with Unkey](https://github.com/djnovin/unkey-rust-actix)
 » 10-October-2024 by Devansh Baghel [Deno Unkey Ratelimit Starter](https://github.com/Devansh-Baghel/deno-unkey-ratelimit-starter)
 » 10-October-2024 by Harsh Bhat [FastAPI Unkey Boilerplate](https://github.com/harshsbhat/unkey-fastapi-boilerplate)
-» 11-October-2024 by Prabin Subedi [Starter template for Unkey Route protecttion in  Hono Webframework in Cloudflare workers ](https://github.com/prabincankod/hono-unkey-cflare)
+» 11-October-2024 by Prabin Subedi [Starter template for Unkey Route protecttion in Hono Webframework in Cloudflare workers ](https://github.com/prabincankod/hono-unkey-cflare)
```

Markdownlint

Spaces inside link text

(MD039, no-space-in-links)